pkg/kubernetes/utils.go

```diff
@@ -42,3 +123,9 @@ func ConvertToDeployment(obj runtime.Object) (*appsv1.Deployment, bool) {
 func UpdateConfigMapObjectValue(configMap *corev1.ConfigMap, key string, value string) {
 	configMap.Data[key] = value
 }
+
+func init() {
+	if os.Getenv("MODE") == "test" {
```

Comment:
I think we can set the env variable in the `cmd/utils/flags.go`!
Or, we can set the directory of manifest in env. (`KUBERNETES_MANIFEST_DIR`) And we can pass it making config filed in Kubernetes.
I think our working directory is `tokamak-titond-backend` and we have to set `./testdata`. I think you executed this code in `cmd/titond`. If it is correct, we have to change this path. Commonly, project root directory is used working directory. And we can run like this `go run cmd/titond/main.go`

Reply:
thank you! i fixed it.